.github/workflows/announcements.yml

```diff
@@ -0,0 +1,35 @@
+name: Fetch announcements from issue
+on: issue_comment
+jobs:
+  issue_commented:
+    # This job only runs for issue comments
+    name: Update Announcements
+    if: ${{ (!github.event.issue.pull_request) && github.event.issue.number == 181 }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - uses: actions/cache@v2
+        name: configure pip caching
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/python_requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install Python dependencies
+        run: |
+          python -m pip install -r .github/python_requirements.txt
+      - name: Run update_announcements script
+        run: |
+          python ./.github/actions/update_announcements.py
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        name: Commit and push
+        with:
+          commit_message: "Updated Announcements"
+          file_pattern: src/*.js src/*.json
```

Comment:
Again, not entirely sure if there are security risks, but since we are force pushing, to follow the principle of least privilege, let's change this file pattern to be limited to only the announcements data file.

Reply:
Yes, for now, not sure what the format will be like, I guess I will make it a json for now